pkg/authorization/authz_unix_test.go

```diff
@@ -173,129 +174,127 @@ func TestDrainBody(t *testing.T) {
-	var (
-		testcases = []struct {
-			url         string
-			contentType string
-			expected    bool
-		}{
-			{
-				contentType: "application/json",
-				expected:    true,
-			},
-			{
-				contentType: "Application/json",
-				expected:    true,
-			},
-			{
-				contentType: "application/JSON",
-				expected:    true,
-			},
-			{
-				contentType: "APPLICATION/JSON",
-				expected:    true,
-			},
-			{
-				contentType: "application/json; charset=utf-8",
-				expected:    true,
-			},
-			{
-				contentType: "application/json;charset=utf-8",
-				expected:    true,
-			},
-			{
-				contentType: "application/json; charset=UTF8",
-				expected:    true,
-			},
-			{
-				contentType: "application/json;charset=UTF8",
-				expected:    true,
-			},
-			{
-				contentType: "text/html",
-				expected:    false,
-			},
-			{
-				contentType: "",
-				expected:    false,
-			},
-			{
-				url:         "nothing.com/auth",
-				contentType: "",
-				expected:    false,
-			},
-			{
-				url:         "nothing.com/auth",
-				contentType: "application/json;charset=UTF8",
-				expected:    false,
-			},
-			{
-				url:         "nothing.com/auth?p1=test",
-				contentType: "application/json;charset=UTF8",
-				expected:    false,
-			},
-			{
-				url:         "nothing.com/test?p1=/auth",
-				contentType: "application/json;charset=UTF8",
-				expected:    true,
-			},
-			{
-				url:         "nothing.com/something/auth",
-				contentType: "application/json;charset=UTF8",
-				expected:    true,
-			},
-			{
-				url:         "nothing.com/auth/test",
-				contentType: "application/json;charset=UTF8",
-				expected:    false,
-			},
-			{
-				url:         "nothing.com/v1.24/auth/test",
-				contentType: "application/json;charset=UTF8",
-				expected:    false,
-			},
-			{
-				url:         "nothing.com/v1/auth/test",
-				contentType: "application/json;charset=UTF8",
-				expected:    false,
-			},
-			{
-				url:         "www.nothing.com/v1.24/auth/test",
-				contentType: "application/json;charset=UTF8",
-				expected:    false,
-			},
-			{
-				url:         "https://www.nothing.com/v1.24/auth/test",
-				contentType: "application/json;charset=UTF8",
-				expected:    false,
-			},
-			{
-				url:         "http://nothing.com/v1.24/auth/test",
-				contentType: "application/json;charset=UTF8",
-				expected:    false,
-			},
-			{
-				url:         "www.nothing.com/test?p1=/auth",
-				contentType: "application/json;charset=UTF8",
-				expected:    true,
-			},
-			{
-				url:         "http://www.nothing.com/test?p1=/auth",
-				contentType: "application/json;charset=UTF8",
-				expected:    true,
-			},
-			{
-				url:         "www.nothing.com/something/auth",
-				contentType: "application/json;charset=UTF8",
-				expected:    true,
-			},
-			{
-				url:         "https://www.nothing.com/something/auth",
-				contentType: "application/json;charset=UTF8",
-				expected:    true,
-			},
-		}
-	)
+	testcases := []struct {
```

Comment:
Oh! This must've been `gofumpt` kicking in; let me know if you want me to revert this part of the diff.